app/controllers/foreman_virt_who_configure/api/v2/configs_controller.rb

```diff
@@ -0,0 +1,121 @@
+
+        api :GET, '/configs', N_("List of virt-who configurations")
+        api :GET, "/organizations/:organization_id/configs", N_("List of virt-who configurations per organization")
+        param_group :search_and_pagination, ::Api::V2::BaseController
+
+        def index
+          @configs = resource_scope_for_index
+        end
+
+        api :GET, "/configs/:id/", N_("Show a virt-who configuration")
+        param :id, :identifier, :required => true
+
+        def show
+        end
+
+        api :GET, "/configs/:id/deploy_script/", N_("Renders a deploy script for the specified virt-who configuration")
+        param :id, :identifier, :required => true
+        def deploy_script
+          respond_to do |format|
+            format.text { render :text => @config.virt_who_bash_script }
+            format.sh { render :text => @config.virt_who_bash_script }
+            format.json
+          end
+        end
+
+        def_param_group :config do
+          param :foreman_virt_who_configure_config, Hash, :required => true, :action_aware => true do
+            param :name, String, :desc => N_("Configuration name"), :required => true
+            param :interval, Config::AVAILABLE_INTERVALS.keys.map(&:to_i), :desc => N_("Configuration interval in minutes"), :required => true
+            param :filtering_mode, Config::FILTERING_MODES.keys.map(&:to_i), :desc => N_("Hypervisor filtering mode, %{unlimited} means no filtering, %{whitelist} means whitelist, %{blacklist} means blacklist") % { :unlimited => Config::UNLIMITED, :whitelist => Config::WHITELIST, :blacklist => Config::BLACKLIST }, :required => false
+            param :whitelist, String, :desc => N_("Hypervisor whitelist, applicable only when filtering mode is set to 1"), :required => false
+            param :blacklist, String, :desc => N_("Hypervisor blacklist, applicable only when filtering mode is set to 2"), :required => false
+            param :hypervisor_id, Config::HYPERVISOR_IDS, :desc => N_("Hypervisor identification"), :required => true
+            param :hypervisor_type, Config::HYPERVISOR_TYPES.keys, :desc => N_("Hypervisor type"), :required => true
+            param :hypervisor_server, String, :desc => N_("Hypervisor server"), :required => true
+            param :hypervisor_username, String, :desc => N_("Hypervisor username"), :required => true
+            param :hypervisor_password, String, :desc => N_("Hypervisor password, required for all hypervisor types except for libvirt"), :required => false
+            param :satellite_url, String, :desc => N_("Satellite server FQDN"), :required => true
+            param :debug, :bool, :desc => N_("Enable debugging output")
+            param :proxy, String, :desc => N_('HTTP Proxy'), :required => true
+            param :no_proxy, String, :desc => N_("Ignore proxy")
+            param :organization_id, Integer, :required => true, :desc => N_("Scope by organizations") if ::SETTINGS[:organizations_enabled]
```

Comment:
Probably copy-paste issue. "Scope by organizations" doesn't seem correct in this context.